girderformindlogger/api/v1/user.py

```diff
@@ -349,8 +349,8 @@ def setUserRelationship(self, id, rel, otherId, otherName):
 
     @access.public(scope=TokenScope.USER_INFO_READ)
     @autoDescribeRoute(
-        Description('Update a user\'s ID Code.')
-        .param('id', 'Profile ID', required=True)
+        Description('Add a new ID Code to a user.')
+        .modelParam('id', model=UserModel, level=AccessType.ADMIN)
```

Comment:
[`ProfileModel`](https://github.com/henryrossiter/mindlogger-app-backend/blob/fb15cb391e0b6c632388c53305e91931cdbba802/girderformindlogger/api/v1/user.py#L18) not `UserModel`. And `AccessType.ADMIN` is probably too strict.